web/webapp/src/main/java/no/nav/foreldrepenger/melding/sikkerhet/pdp/XacmlRequestBuilderTjenesteImpl.java

```diff
@@ -54,13 +54,18 @@ private XacmlAttributeSet byggRessursAttributter(PdpRequest pdpRequest, Tuple<St
         XacmlAttributeSet resourceAttributeSet = new XacmlAttributeSet();
         resourceAttributeSet.addAttribute(NavAttributter.RESOURCE_FELLES_DOMENE, pdpRequest.getString(NavAttributter.RESOURCE_FELLES_DOMENE));
         resourceAttributeSet.addAttribute(NavAttributter.RESOURCE_FELLES_RESOURCE_TYPE, pdpRequest.getString(NavAttributter.RESOURCE_FELLES_RESOURCE_TYPE));
+        setOptionalValueinAttributeSet(resourceAttributeSet, pdpRequest, NavAttributter.RESOURCE_FORELDREPENGER_SAK_SAKSSTATUS);
```

Comment:
Ønsker ikke å benytte oss av NavAttributter lenger. Se fpsak for korrekte klasser. Får riktige avhengigheter om du oppdaterer til siste versjon av felles.

Reply:
Det er jo helt likt i fpsak mater?

Reply:
Tenkte på felles :| 

https://github.com/navikt/fp-felles/blob/22791e96742ad86cc3285a4266e0cf16a5b54f1a/felles/sikkerhet/sikkerhet/src/test/java/no/nav/vedtak/sikkerhet/pdp/XacmlRequestBuilderTjenesteImpl.java